packages/ui/src/breadcrumbs.tsx

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable react/display-name */
+import { Box, PropsWithBox, Text } from "@saleor/macaw-ui/next";
+import Link from "next/link";
+import React from "react";
+import styles from "./breadcrumbs.module.css";
+
+export type BreadcrumbsItemProps = PropsWithBox<{
+  href?: string;
+  active?: boolean;
+  separator?: React.ReactNode;
+}>;
+
+const BreadcrumbsItem = ({
+  children,
+  href,
+  separator = ">",
```

Comment:
In Macaw there is a chevron icon, maybe its better? ASCII symbols are often misplaced

Reply:
Also are you sure this is part of the *item*? I think its part of the container. I don't want to have this prop in every item to overwrite it